stunnel/stunnel.ml

```diff
@@ -134,21 +135,25 @@ let config_file verify_cert extended_diagnosis host port =
     ]
   ; if is_fips then ["fips=yes"] else ["fips=no"]
   ; if extended_diagnosis then ["debug=authpriv.7"] else ["debug=authpriv.5"]
+  ; [ "sslVersion = TLSv1.2"
+    ; "ciphers = " ^ Xcp_const.good_ciphersuites
+    ; "curve = secp384r1"
+    ]
```

Comment:
Although I find defining ssl version and ciphers here strange, it makes sense. I can't figure why is the curve parameter needed here.